packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -371,41 +394,43 @@ class InboundReferencesWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return ValueListenableBuilder<InboundReferences?>(
       valueListenable: inboundReferences,
-      builder: (context, inboundReferences, child) {
+      builder: (context, inboundReferences, _) {
+        final references = inboundReferences == null
+            ? <Widget>[]
+            : _inboundReferencesList(context, inboundReferences);
+
         return VmExpansionTile(
           title: 'Inbound References',
           onExpanded: onExpanded,
           children: inboundReferences == null
```

Comment:
Same comment as above, this can be rewritten into a single list with conditions.